backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -343,15 +344,14 @@ private async void OnBatteryUpdate(object? sender, MqttReceivedArgs mqttArgs)
             else
             {
                 await robotService.UpdateRobotBatteryLevel(robot.Id, batteryStatus.BatteryLevel);
-                await timeseriesService.Create(
-                    new RobotBatteryTimeseries
-                    {
-                        MissionId = robot.CurrentMissionId,
-                        BatteryLevel = batteryStatus.BatteryLevel,
-                        RobotId = robot.Id,
-                        Time = DateTime.UtcNow
-                    }
-                );
+
+                try { await timeseriesService.AddBatteryEntry(robot.CurrentMissionId!, batteryStatus.BatteryLevel, robot.Id); }
+                catch (NpgsqlException e)
```

Comment:
Is the commit message wrong? Should NplsqlException correspond to NpgsqlException?

Reply:
Commit message is wrong, will fix it.